crates/geometry/src/line.rs

```diff
@@ -17,6 +17,7 @@ pub struct Line<Frame, const DIMENSION: usize>(
 );
 
 pub type Line2<Frame> = Line<Frame, 2>;
+pub type Line3<Frame> = Line<Frame, 3>;
```

Comment:
Is this used anywhere?

Reply:
No, just there for completeness ;) Did not hurt, and the next person is lucky